controllers/news.js

```diff
@@ -17,15 +17,15 @@ const getNews = async (req, res, next) => {
         message: result,
```

Comment:
```suggestion
        message: "mensaje",
        body: result
```
devolver el resultado en el body

Reply:
Modificado